src/dctest/expressions.cljs

```diff
@@ -83,8 +83,17 @@ IDENTIFIER_PART  ::= [a-zA-Z0-9] | '$' | '_'
 (def parser
   (ebnf/Grammars.W3C.Parser. grammar))
 
-(defn get-ast [text]
-  (->clj (.getAST parser text "InterpolatedText")))
+(defn read-ast [text & [start]]
+  (let [start (or start "Expression")]
+    (->clj (.getAST parser text start))))
+
+(defn print-ast [ast]
```

Comment:
Is this still considered an AST?  Note, there's also a `pprint-ast` that actually takes the ebnf-returned AST.  I don't feel strongly in any direction, just noting the potential confusion.

Reply:
No, you're right. I've changed them to `print-obj*`.